index.js

```diff
@@ -26,94 +27,52 @@ class CumulusMessageAdapterExecutionError extends Error {
 async function generateCMASpawnArguments(command) {
   const adapterDir = process.env.CUMULUS_MESSAGE_ADAPTER_DIR || './cumulus-message-adapter';
   const systemPython = await lookpath('python');
-  if (systemPython) {
+  if (systemPython && process.env.USE_CMA_BINARY !== 'true') {
     return [systemPython, [`${adapterDir}`, command]];
   }
   // If there is no system python, attempt use of pre-packaged CMA binary
-  return [`${adapterDir}/cma`, [command]];
+  return [`${adapterDir}/cma_bin/cma`, [command]];
```

Comment:
I thought this would be `/cma` based on this note:

https://github.com/nasa/cumulus-message-adapter/pull/74/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR19

Reply:
It should be, will fix

Reply:
Er, wait, no, this is correct.     This *is* the client update to /cma_bin - the client library is handling the change to the CMA here so it's transparent to end users, who still just need the environment file to point to the unpacked zip.